pyrdp/mitm/RDPMITM.py

```diff
@@ -238,6 +239,11 @@ def doClientTls(self):
 
         # Add unknown packet handlers.
```

Comment:
I think that comment needs to be revisited. It's just confusing now and since we parse NTLMSSP better we can rewrite it to be better here